tomcat/build_test.go

```diff
@@ -381,9 +370,22 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 			}
 			ctx.StackID = "test-stack-id"
 
+			Expect(os.Setenv("BP_TOMCAT_EXT_CONF_SHA256", "test-sha256")).To(Succeed())
```

Comment:
This is minor, but I have been trying to migrate from manually calling `os.Setenv` to using `t.Setenv`. The latter will automatically clean up after the test, so it's less verbose and doesn't require manual clean up.

https://pkg.go.dev/testing#B.Setenv

If you could update the tests here to do that, it would be appreciated.